tutorials/llm/slimpajama/data_pipeline.ipynb

```diff
@@ -0,0 +1,328 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data Processing for NeMo 2.0 LLMs\n",
+    "\n",
+    "This tutorial will cover the steps to go from a raw pretraining dataset all the way to configuring the data module for pretraining using a NeMo 2.0 recipe.\n",
+    "We will be using the [SlimPajama-627B](https://huggingface.co/datasets/cerebras/SlimPajama-627B>) dataset for reference. We will also show how to exclude certain sources from the dataset, for instance, by default we will be excluding all data from the `RedPajamaBook` set.\n",
+    "\n",
+    "This tutorial involves four steps:\n",
+    "\n",
+    "1. Downloading data\n",
+    "2. Extracting data\n",
+    "3. Concatenating data\n",
+    "4. Preprocessing data for NeMo 2.0/Megatron\n",
+    "\n",
```

Comment:
change to an imperative verb , fix punctuation, use Megatron Core name (if correct)

4. Preprocess data for NeMo 2.0/Megatron Core.